usl_models/usl_models/flood_ml/batch_predictor.py

```diff
@@ -0,0 +1,191 @@
+"""Utilities for running batch predictions on files stored in GCS."""
+
+import json
+import urllib.parse
+import logging
+import dataclasses
+import tensorflow as tf
+
+from google.cloud import firestore  # type:ignore[attr-defined]
+from google.cloud import storage  # type:ignore[attr-defined]
+import numpy as np
+
+from usl_models.flood_ml import model
+from usl_models.flood_ml import prediction_dataset
+
+
+@dataclasses.dataclass
+class BatchPredictor:
+    """Runs batch predictions on files stored in GCS.
+
+    Ouputs data based on the specified `output_bucket`, `model_id`,
+    and `run_id`.
+    """
+
+    db: firestore.Client
+    client: storage.Client
+    output_bucket: str
+    model_id: str
+    model: model.FloodModel
+    run_id: str
+    batch_size: int = 1
```

Comment:
Are we able to default to a bigger batch size than 1?

Reply:
Sure, added a note for how to tune this.